driver/node.go

```diff
@@ -378,19 +378,19 @@ func getBlockDeviceByDiskID(ctx context.Context, diskID string) (dev string, err
 
 // udevWaitDiskToSettle uses udevadm to wait events in event queue to be handled.
 func udevWaitDiskToSettle(ctx context.Context, path string) error {
-	return exec.CommandContext(ctx,
+	return exec.CommandContext(ctx, //nolint: gosec // TODO: should we validate path that might not exists? Disabled for now
```

Comment:
No need to check for udevadm availability, but no harm in using the full path if it helps with gosec.

Reply:
I'll looking to that after this PR is merged. I tried to add some validation to that path variable but that didn't help. Also tried to use short options so that `fmt.Sprintf` wasn't needed but that didn't help either.